Rakefile

```diff
@@ -335,3 +335,25 @@ task :fix_line_endings do
     end
   end
 end
+
+desc <<-DESC
+  Scale a dataset to amount_of_residences
+
+  Arguments:
```

Comment:
This documentation is a little out-of-date now:

* DATASET → FULL_DATASET
* LOCAL_DATASET → DERIVED_DATASET

It would be nice if the task could do some basic error checking – that values were provided for all three options and the full dataset exists – because the default error isn't going to be very helpful to researchers:

```
rake aborted!
Could not find a fulldataset with the key nil
```